Examples/test-suite/member_funcptr_galore.i

```diff
@@ -95,6 +95,9 @@ double (Space::Shape::*abcvar)(Thing<short>, Thing< const Space::Shape * >[]) =
 %constant double (Space::Shape::*AREAPT)(Space::Shape &, int & (FunkSpace::Funktions::*)(const int &, int)) = &Space::Shape::area;
 %constant double (Space::Shape::*AREAPT_CONST)(Space::Shape &, int & (FunkSpace::Funktions::*)(const int &, int) const) = &Space::Shape::area_const;
 %constant double (Space::Shape::*PERIMPT)(Thing<short>, Thing< const Space::Shape * >[]) = &Space::Shape::abc;
+#ifdef SWIGFORTRAN
+%noparameter NULLPT;
```

Comment:
I think this is masking a problem with function pointer handling for Fortran. See http://swig.org/Doc3.0/SWIG.html#SWIG_nn30 (just %constant is required). In which case, put this in the set of broken tests instead of using %noparameter until fixed.

Reply:
Done